_docs/quickstarts/scanner/default-rules.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Default Rules
+order: 106
+---
+
+# {{ page.title }}
+
+Forseti Scanner is preconfigured with some default rules. When any of these rules' conditions are met, 
+Scanner will create a [violation]({% link _docs/quickstarts/scanner/policy-violations.md %}).
+
+* BigQuery
+  * Datasets should not be public.
+  * Datasets should not be accessible by users who are @gmail.com.
+  * Datasets should not be accessible by groups who are @gmail.com.
+ 
+* Cloud Storage
+  * Buckets should not be publicly accessible (“AllUsers”).
+  * Buckets should not be accessible by any authenticated user (“AllAuthenticatedUsers”).
+ 
+* Cloud SQL
+  * Cloud SQL instances should not allow access from anywhere (authorized networks).
+  * Cloud SQL instances should not allow access over SSL from anywhere (authorized networks).
+ 
+* G Suite Groups
+  * My company users (@domain.tld) and all gmail users are allowed to be members of my G Suite groups.
```

Comment:
We should say "Your" instead of "my" - this relates the text to the user more directly. E.g., "Your company users... are allowed to be members of your G Suite groups." (all occurrences through the doc)